coral/src/app/router.tsx

```diff
@@ -13,6 +14,10 @@ const routes: Array<RouteObject> = [
     path: "/topics",
     element: <Topics />,
   },
+  {
+    path: "/topics/request",
```

Comment:
Excellent point! What do you think, if we would keep the route as is, but we would remove the navigation possibility from the application. I recall us having some discussion over feature flags, did that yield into any code?

Reply:
Initial investigation resulted in my take that we should not add the route to the router until feature is functional, but as I said, we have tests for navigation :/ Not very attached to my initial take ^^ Maybe a change to `PreviewBanner` could be made so that it renders a "Danger" kind of banner instead of "Info", specifying that this part is not functional.

Also that we should have "dev" routes so that navigation when running project locally doesn't break:

![Screenshot 2022-12-21 at 09 40 39](https://user-images.githubusercontent.com/20607294/210319862-7d1a352e-ae19-4348-bee8-efeda5c9bb45.png)

Planned to do a PR for this but got absorbed by the critical path tasks.